### ISSUE TITLE
Release Che 7.9.0

### ISSUE THREAD

**Issue by nickboldt** [eclipse-che/che#16008]
### List of pending issues / PRs

- [x] https://github.com/che-incubator/kubernetes-image-puller/pull/7
- [x] LS not working for Java Maven #16051
- [ ] *(add blockers here)*

### Release status

In parallel, the following releases can be done:

* che-theia,
* che-machine-exec,
* che-devfile registry, then che-plugin-registry (once che-theia and machine-exec are done)
* che-parent, then che-docs, then che as Release Candidate for QE

Then in series:

* che-parent, then che-docs, then che as Release
* che-operator, then chectl

- [x] che-theia _*(1eaecd5cbf93468a74988ddcd347402fcbf14cbe)*_
- [x] che-machine-exec
- [x] che-plugin-registry _(depends on che-theia, che-machine-exec)_
- [x] che-devfile-registry
- [x] che-parent
- [x] che-docs _(depends on parent)_
- [x] che _(depends on all of the above)_
- [x] che-operator _(depends on all of the above)_
  - [x] Kubernetes community operator PR  https://github.com/operator-framework/community-operators/pull/1267
  - [x] OpenShift community operator PR https://github.com/operator-framework/community-operators/pull/1268
- [x] chectl _(depends on all of the above)_
- [x] complete current milestone
  - [x] move incomplete *deferred* issues to backlog
  - [x] move incomplete *WIP* issues to next milestone
  - [x] close completed issues
  - [x] close milestone

| Owner | Process | Script | CI | Artifact(s) |
| --- | --- | --- | --- | --- |
| @azatsarynnyy| [che-theia](https://github.com/eclipse/che-theia/blob/master/RELEASE.md) | [***MANUAL***](https://github.com/eclipse/che-theia/blob/master/RELEASE.md) | [centos](https://ci.centos.org/job/devtools-che-theia-che-release/) | [`quay.io/eclipse/che-theia`](https://quay.io/eclipse/che-theia) |
| @nickboldt| [che-machine-exec](https://github.com/eclipse/che-machine-exec/blob/master/RELEASE.md) | [make-release.sh](https://github.com/eclipse/che-machine-exec/blob/master/make-release.sh) | [centos](https://ci.centos.org/job/devtools-che-machine-exec-release/) | [`quay.io/eclipse/che-machine-exec`](https://quay.io/eclipse/che-machine-exec)| 
| @nickboldt| [che-devfile-registry](https://github.com/eclipse/che-devfile-registry/blob/master/RELEASE.md) | [make-release.sh](https://github.com/eclipse/che-devfile-registry/blob/master/make-release.sh) | [centos](https://ci.centos.org/job/devtools-che-devfile-registry-release/) | [`quay.io/eclipse/che-devfile-registry`](https://quay.io/eclipse/che-devfile-registry)| 
| @nickboldt / @ericwill| [che-plugin-registry](https://github.com/eclipse/che-plugin-registry/blob/master/RELEASE.md) | [make-release.sh](https://github.com/eclipse/che-plugin-registry/blob/master/make-release.sh) | [centos](https://ci.centos.org/job/devtools-che-plugin-registry-release/) | [`quay.io/eclipse/che-plugin-registry`](https://quay.io/eclipse/che-plugin-registry)| 
| @vparfonov / @mkuznyetsov| [che-parent](https://github.com/eclipse/che/blob/master/RELEASE.md) | [***MANUAL***](https://github.com/eclipse/che/blob/master/RELEASE.md) | [centos](https://ci.centos.org/job/devtools-che-parent-che-release/) | [che-parent](https://search.maven.org/search?q=a:che-parent), [che.depmgt](https://search.maven.org/artifact/org.eclipse.che.depmgt/maven-depmgt-pom) |
| @vparfonov / @mkuznyetsov| [che-docs](https://github.com/eclipse/che/blob/master/RELEASE.md) | [***MANUAL***](https://github.com/eclipse/che/blob/master/RELEASE.md) | [centos](https://ci.centos.org/job/devtools-che-docs-che-release/) | [che-docs](https://search.maven.org/search?q=a:che-docs)
| @vparfonov / @mkuznyetsov| [che](https://github.com/eclipse/che/blob/master/RELEASE.md) | [***MANUAL***](https://github.com/eclipse/che/blob/master/RELEASE.md) | [centos](https://ci.centos.org/job/devtools-che-che-release/) | [che.core](https://search.maven.org/search?q=che.core), [`quay.io/eclipse/che-server`](https://quay.io/eclipse/che-server) |
| @tolusha| [che-operator](https://github.com/eclipse/che-operator/blob/master/RELEASE.md) | [make-release.sh](https://github.com/eclipse/che-operator/blob/master/make-release.sh) | ? | [`quay.io/eclipse/che-operator`](https://quay.io/eclipse/che-operator)| 
| @tolusha| [chectl](https://github.com/che-incubator/chectl/blob/master/RELEASE.md) | [make-release.sh](https://github.com/che-incubator/chectl/blob/master/make-release.sh) | [travis](https://travis-ci.org/che-incubator/chectl) | [chectl releases](https://github.com/che-incubator/chectl/releases)

**Comment by davidfestal** [comment]
@nickboldt I think that the Che operator release is now owned by @tolusha.
@tolusha can you confirm ?

**Comment by tolusha** [comment]
@davidfestal Sure

**Comment by nickboldt** [comment]
Fixed above and in template too https://github.com/eclipse/che/pull/16037 so it's correct for 7.10

**Comment by tolusha** [comment]
@mkuznyetsov let me know when che is released

**Comment by nickboldt** [comment]
Che 7.9.0 plugin registry re-released.

**Comment by tolusha** [comment]
`chectl` has been released.

**Comment by nickboldt** [comment]
how is chectl done but che operator not?

**Comment by nickboldt** [comment]
also, where does the che-operator release build run? localhost ? ci-centos?

**Comment by tolusha** [comment]
`che-operator` has been released as well.
The release process is run on localhost only.

**Comment by nickboldt** [comment]
:+1:  as long as you're iterating towards a process that can be run on a shared host, I'm happy :D
